ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml

```diff
@@ -37,13 +39,21 @@ spec:
           exit 1
         fi
 
-        CI_FILE_PATH="$(params.operator_path)/ci.yaml"
+        IFS=',' read -ra operator_array <<< "$(params.affected_operators)"
```

Comment:
This section would deserves a comment on why we need to iterate over the list in reverse order.